src/styles/main.scss

```diff
@@ -1,3 +1,68 @@
-body {
+@import './variables';
+
+.body {
   margin: 0;
+  height: 100vh;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  counter-reset: section;
 }
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  width: 706px;
+  gap: 1px;
+
+
+  &__day {
+    box-sizing: border-box;
+    background-color: $cell-color;
+    width: 100px;
+    height: 100px;
+    border: 1px solid black;
+    position: relative;
+    transition: all 0.5s;
```

Comment:
https://stackoverflow.com/questions/8947441/css3-transitions-is-transition-all-slower-than-transition-x

better to avoid the transition all